aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -712,6 +714,11 @@ std::tuple<Tensor&, Tensor&> frexp_out(const Tensor& self,
   return std::tuple<Tensor&, Tensor&>(mantissa, exponent);
 }
 
+// alias for lgamma, implements special.gammanln equivalent to
```

Comment:
Were these changes picked up from another PR?

Reply:
These changes were caused by the code conflicts. At that time, `UnaryOps.cpp` encountered the code conflicts between `torch.frexp` and `torch.special_gammaln`, I merged the latest code from `master` branch and resolved the conflicts in the merge commit 54472cc1f26271c56ad0f23abf01d490212ac0a1.

The above changes wouldn't show up in the final changes of this PR, you may confirm this via the `Files Changed` tab of Github (https://github.com/pytorch/pytorch/pull/51097/files)

![image](https://user-images.githubusercontent.com/40605299/110806006-9d552e80-82bc-11eb-8afd-4bd35b398727.png)

The above diff shows only the changes of `torch.frexp` are made to `UnaryOps.cpp`.

Reply:
You are right, thank you for clarifying this.